src/handlers/add-comment.ts

```diff
@@ -81,3 +103,5 @@ export async function addCommentToIssue(context: Context, message: string, optio
     throw error;
   }
 }
+
+addCommentToIssue.lastCommentId = undefined as number | undefined;
```

Comment:
```suggestion
addCommentToIssue.lastCommentId = null;
```

Setting to undefined seems really unusual. Why not null

Reply:
https://github.com/ubiquity-os-marketplace/command-ask/pull/58/commits/e4fd90e25c2db5b48881705ce3d3f16bdf011b31